rps.js

```diff
@@ -0,0 +1,37 @@
+///game :(
+
+function game(user) {
+
+
+    const pcNum = function () {
+        let pc = Math.floor(Math.random() * 3)
+        if (pc < 1) {
+            return "rock"
+
+        } else if (pc > 1) {
+            return "paper"
+        } else {
+            return "scissors"
+        }
+    }
+    
+    if (user === "rock " || user === "paper" || user === "scissers") {
```

Comment:
Great solution, But you just have two issues in this line:
1- ```user === "rock "``` adding space after rock, space has value, So ```"rock "``` does not equal to ```"rock"```